### PR TITLE
Kp 7226 fetch ms info via oai pmh

### PR DESCRIPTION
**Pull request by kaisakui**
This PR contains functionality to fetch all data from Metashare using OAI-PMH protocol.

### CODE REVIEW

File: harvester/pmh_interface.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    def get_all_metadata_records(self):
+        """
+        Get content for all records available from the API.
+        """
+        records = self.sickle.ListRecords(metadataPrefix="info")
+        metadata_records_list = []
+        for record in records:
+            metadata_records_list.append(record)
+        return metadata_records_list
```
Comment (helhotti) [inline harvester/pmh_interface.py]: Is it necessary/a good idea to convert the generator that sickle returns to a list? You probably know best what kind of purposes this function will be used for.

If a list is what you want to return, you can convert the result of `ListRecords` to a list like this:

```
records = list(self.sickle.ListRecords(metadataPrefix="info"))
```
or alternatively only wrap into a list when returning:
```
return list(records)
```

So there's no need for a loop.

**Review by helhotti — CHANGES_REQUESTED** [review]
One python-related comment, otherwise this seems to do what is asked!

File: harvester/pmh_interface.py (hunk 1 above)
Comment (helhotti) [reply harvester/pmh_interface.py]: I just realized that the data type that sickle returns is not a normal generator after all, so my previous comment is pretty much useless :D unless you want to actually return a generator, in which case you could use `yield` in the loop. But if list is what is wanted, this is fine.

Comment (kaisakui) [reply harvester/pmh_interface.py]: Good point! Thanks

**Review by helhotti — APPROVED** [review]
Looks good :+1: I think the generator route may be more future-proof, you can always turn that into a list if needed.

**Comment by mmatthiesencsc** [comment]
I hoped this to be simple, thanks!

**Pull request merged**